public/stylesheets/local.css

```diff
@@ -114,6 +114,14 @@ pre {
     overflow-wrap: normal;
 }
 
+/* MathJax SVG shape rendering: Prefer accurate antialiased font shapes
+   over "crisp" pixel-aligned ones. This may be less sharp in some cases,
+   but it can prevent thin lines from disappearing. */
```

Comment:
Is there a MathJax issue tracking this? If so, let's link to it so we can identify when this specific CSS won't be necessary in the future. If not, please file one with them!

Reply:
I can try to raise it to them as an issue, but it seems like it's a limitation of the renderer on some platforms that they don't have control over. I could suggest that they try adding this CSS to their own styles, or maybe PR that if I can find the right place.

Reply:
In case it _is_ something they have control over, I still think it would be a good idea to report this upstream. If they really can't do anything about it, they may still want to document this limitation and encourage folks to use the SVG renderer. (I acknowledge that this doesn't help us, but it feels like the right thing to do.)